grpc-native/src/main/java/org/ballerinalang/net/grpc/builder/syntaxtree/utils/ClientUtils.java

```diff
@@ -355,6 +370,16 @@ private static Function getReceiveContextFunction(Method method) {
                         "toString",
                         new String[]{}
                 );
+            } else if (isTimestamp(method.getOutputType())) {
```

Comment:
Since the `null` check is already done, we can simply do a string comparison here without calling the `isTimestamp` function.

Reply:
fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/312/commits/7542e0e21e09589aa10e28ae97f881c3ccc4d7a2